### DISCUSSION TITLE
Add a getNextPage thing in infinite-query

### DISCUSSION THREAD

**Discussion by CaffeineDuck** [TanStack/query#1870] [Ideas]
Hey I have a suggestion, in react-query infinite queries maybe a getNextPage things can be added cause getNextParam is helpful but sometimes the API provides us directly with the next page link so it would be helpful if we could directly send the next page link instead of passing the param!

**Comment by TkDodo** [comment]
param is just a name. If your api returns a full url, you can return that from getNextParam as well and then use that value to make the request
